package.json

```diff
@@ -4,8 +4,12 @@
   "private": true,
   "homepage": ".",
   "dependencies": {
+    "@craco/craco": "^7.0.0-alpha.0",
```

Comment:
This is the beta version module. If there is a new version supporting CRA 5, we should update it.